src/backend/controllers/userController.js

```diff
@@ -78,7 +78,7 @@ exports.register = async (req: Request, res: Response) => {
 		// Create initial tank for user
 		let tank = new Tank();
 		tank.userId = user.id;
-		tank.tankName = 'Starter Tank';
+		tank.tankName = `${userName}'s Starter Tank`;
```

Comment:
Are users allowed to rename tanks they have already created?

Reply:
It's outside the scope of this PR, but the API to update the tank supports it